backend/dep/wire.go

```diff
@@ -211,3 +210,22 @@ func InjectRoutingService(
 	)
 	return mdservice.Service{}
 }
+
+func getEnv(key, defaultValue string) string {
```

Comment:
These functions already exist https://github.com/short-d/short/blob/master/backend/main.go#L85
We should reduce the number of code duplicates as much as we can.

Reply:
I removed the mustInt function from backend/main and placed an exportable function in dep package. But for getEnv I'm not sure how to do it without creating cycles because we need AUTH_TOKEN_LIFETIME_IN_SEC inside dep package

Reply:
I meant to move initialization to `main.go` and add the corresponding field to the `Config` struct